cypress/integration/team/teammates_pagination_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const uuid = () => Cypress._.random(0, 1e6);
+
+describe('Teams Suite', () => {
+    before(() => {
+        // MaxUsersPerTeam setting must be >60.
+        // Requires more than 60 available test users.
+        // (e.g., make cypress-test-data)
+        Cypress.env('team-size', 60);
+
+        // 1. New test case user
+        cy.loginAsNewUser();
+    });
+
+    it('TS14868 Team Admin can use Next button to page through list in Manage Members', async () => {
+        const job = uuid();
+        const teamDescr = `ts14868-${job}`;
+        const teamKey = `t-${job}`;
+
+        // 2. Create new team for test
+        cy.createNewTeam(teamDescr, teamKey);
```

Comment:
Since we are building data and not doing test for creating a team, it's preferable to create team via API command (`cy.apiCreateTeam`) rather than UI command (`cy.createNewTeam`).  Please see existing specs on how we're using it.

Reply:
I used the existing cy.apiCreateTeam in the today's PR.